include/grpc/grpc_security.h

```diff
@@ -48,6 +48,7 @@ extern "C" {
 
 #define GRPC_X509_CN_PROPERTY_NAME "x509_common_name"
 #define GRPC_X509_SAN_PROPERTY_NAME "x509_subject_alternative_name"
+#define GRPC_X509_CLIENT_CERT_PROPERTY_NAME "x509_client_cert"
```

Comment:
Sorry, this is not only the client cert... the server cert is going to be exposed on the client side. How about x509_peer_cert?